lib/api.js

```diff
@@ -314,6 +320,10 @@ function transformOptions(options, encoding) {
     transformed.windowOptions.pretendToBeVisual = Boolean(options.pretendToBeVisual);
   }
 
+  if (options.ResourceLoader) {
```

Comment:
Why is there also a capital-R `ResourceLoader` option?

Reply:
this is an error